applications/launchpad_v2/package.json

```diff
@@ -63,11 +62,8 @@
   },
   "resolutions": {
     "react": ">=18.0.0",
-    "react-dom": ">=18.0.0"
-  },
-  "overrides": {
-    "react": ">=18.0.0",
-    "react-dom": ">=18.0.0"
+    "react-dom": ">=18.0.0",
+    "nth-check": "^2.0.1"
```

Comment:
what is this used for?

Reply:
npm audit threw a problem with nth-check - one of dependencies were using older version of nth-check, so here I force npm to use newer version. overrides are used by npm, while resolutions by yarn. I'm going to revisit npm audit in te next, bc I have added daily job on Github Actions that runs at 8:00 am.